attributes/master.rb

```diff
@@ -135,6 +135,11 @@
   master['use_system_accounts'] = true
 
   #
+  # The groups of user under which Jenkins is running. Works for runit only.
+  #
+  master['groups'] = [node['jenkins']['master']['group']]
```

Comment:
Since this tuneable is only used by the Runit configuration we should definetely place it in the `runit` namespace: `node['jenkins']['master']['runit']['groups']`.